tests/integration/cloud/rest_contracts/conftest.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+import enum
+import os
+import pathlib
+import subprocess
+from typing import TYPE_CHECKING, Callable, Final, Union
+
+import pytest
+from pact import Consumer, Pact, Provider
+from pact.matchers import Matcher  # noqa: TCH002
+from typing_extensions import Annotated  # noqa: TCH002
+
+from great_expectations.compatibility import pydantic
+from great_expectations.core.http import create_session
+
+if TYPE_CHECKING:
+    from requests import Session
+
+PACT_MOCK_HOST: Final[str] = "localhost"
+PACT_MOCK_PORT: Final[int] = 9292
+PACT_DIR: Final[str] = str(
+    pathlib.Path(pathlib.Path(__file__).parent, "pacts").resolve()
+)
+
+
+class RequestMethods(str, enum.Enum):
+    DELETE = "DELETE"
+    GET = "GET"
+    PATCH = "PATCH"
+    POST = "POST"
+    PUT = "PUT"
```

Comment:
I want to say we get this for free through another library (maybe stdlib) but that could be a later version of Python

Reply:
Yeah it seems stupid, but pydantic works nicely with enums, and we can use it in the tests if desired.

Reply:
One thing to note is there is no canonical list of these methods. It can be any string.